specification/maps/data-plane/readme.md

```diff
@@ -68,7 +86,6 @@ input-file:
   - Microsoft.Maps/Timezone/preview/1.0/timezone.json
   - Microsoft.Maps/Traffic/preview/1.0/traffic.json
   - Microsoft.Maps/Weather/preview/1.0/weather.json
-  - Microsoft.Maps/WFS/preview/2.0/wfs.json
```

Comment:
I believe this removal is why you're seeing the sudden errors for `unreferenced json from readme`. [the first avocado error](https://github.com/Azure/azure-rest-api-specs/pull/26725/checks?check_run_id=19202070457) specifically. I think it's iffy reporting, but all stemming from the unreferenced `wfs.json`. I'm betting it's breaking `oav`'s logic when resolving the swagger itself, which is why the examples are also throwing.

Reply:
I think if you make changes to the readme like this, you're going to need to go through the ARM review process as described in the PR body. (the image with labeling instructions etc in the automatically created body of the PR).